package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "ably-react-native",
   "description": "React Native wrapper for the JavaScript realtime client library for Ably.io, the realtime messaging service",
-  "version": "1.1.1",
+  "version": "1.2.0",
   "dependencies": {
-    "ably": "^1.1.3",
+    "ably": "@1.2.4",
```

Comment:
what does the `@` do in the version specifier? can't see any mention of that in the npm docs

Reply:
yeah not sure where i got that from... fixed now.